accepted/2021/InboxSourceGenerators.md

```diff
@@ -0,0 +1,126 @@
+# Inbox Source Generators
```

Comment:
Hi
I was wondering what Inbox Source Generators is referring to?

Reply:
"Inbox" meaning they ship with the .NET runtime. For example, System.Text.RegularExpressions ships "inbox" in the .NET runtime. If/when we build a source generator for Regex, that source generator would ship along side the System.Text.RegularExpressions library in the .NET runtime.

So "Inbox Source Generators" refers to source generators that ship along with the .NET runtime.

Reply:
I believe this should be sufficiently defined below in scenarios.  I tried some other titles and they all seemed too generic and  implied more than just inbox.  Any suggestions for a different title that doesn't use jargon up front?